projects/cypress/CY8CKIT_064S0S2_4343W/mtb/jitp/config_files/iot_pkcs11_config.h

```diff
@@ -32,9 +32,21 @@
 #ifndef _AWS_PKCS11_CONFIG_H_
 #define _AWS_PKCS11_CONFIG_H_
 
+/**************************************************/
+/******* DO NOT CHANGE the following order ********/
+/**************************************************/
+
+/* Include logging header files and define logging macros in the following order:
+ * 1. Include the header file "logging_levels.h".
+ * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
+ * the logging configuration for MQTT.
+ * 3. Include the header file "logging_stack.h", if logging is enabled for MQTT.
```

Comment:
Copy-pasta fixes across all config files!
```suggestion
 * 2. Define the LIBRARY_LOG_NAME and LIBRARY_LOG_LEVEL macros depending on
 * the logging configuration for PKCS#11.
 * 3. Include the header file "logging_stack.h", if logging is enabled for PKCS#11.
```